README.md

```diff
@@ -47,23 +46,11 @@ tasks.withType(JavaCompile) {
 }
 ```
 
-Let's walk through this script step by step.  The `plugins` section pulls in the [Gradle Error Prone plugin](https://github.com/tbroyer/gradle-errorprone-plugin) for Error Prone integration. If you are using the older `apply plugin` syntax instead of a `plugins` block, the following is equivalent:
-```gradle
-buildscript {
-  repositories {
-    gradlePluginPortal()
-  }
-  dependencies {
-    classpath "net.ltgt.gradle:gradle-errorprone-plugin:0.6"
-  }
-}
-
-apply plugin: 'net.ltgt.errorprone'
```

Comment:
I presume this is no longer supported?

Reply:
So I was following the style of other READMEs I've seen recently, including the Gradle Error Prone plugin.  None of them even document this possibility anymore; it was outdated years ago.